stronk/errors/errors.py

```diff
@@ -4,3 +4,7 @@ class RevokedIdTokenError(Exception):
 
 class InvalidIdTokenError(Exception):
     pass
+
+
+class MissingAuthToken(Exception):
```

Comment:
Should we use [HttpException](https://werkzeug.palletsprojects.com/en/1.0.x/exceptions/#werkzeug.exceptions.HTTPException) instead?